packages/tools/devtools/devtools-browser-extension/public/manifest.json

```diff
@@ -3,7 +3,7 @@
 	"name": "Fluid Framework Developer Tools (Preview)",
 	"description": "Devtools extension for viewing live data about your Fluid application.",
 	"author": "Microsoft",
-	"version": "0.0.3",
```

Comment:
Since this is going to be the first public facing (preview) release, let's make this `0.1.0`. I'll update the wiki docs to reflect the new status / new rules for bumping versions. From here on out, we should basically follow [SemVer](https://semver.org/).